src/StackAdmin/Azs.Compute.Admin/Help/Add-AzsPlatformImage.md

```diff
@@ -32,179 +32,178 @@ Add a new platform image.
 
```

Comment:
There should not be a need to change this file as there is no change in the cmdlet Add-AzsPlatformImage?

Reply:
yes, there might be soem update in platyps script, that caused this change, i will remove.